### ISSUE TITLE
[Request] Upvoting Chatmessages

### ISSUE THREAD

**Issue by KenavR** [larvalabs/breaker#27]
This feature was requested by multiple people (theplanetofthecrepes, TurboooTurtle, apercots). I guess this should be discussed.

**Comment by rickhanlonii** [comment]
Maybe the better route here is to add Slack/Facebook style reactions?

**Comment by megamattron** [comment]
Yeah, I agree with you @rickhanlonii - I think that's the real solution.
